src/components/AdditionalWeatherData/index.js

```diff
@@ -1,34 +1,41 @@
-import React from "react";
-import styled from "styled-components";
-import { WeatherData } from "../WeatherData/Index";
-import { useTranslation } from "react-i18next";
+import React from 'react';
+import styled from 'styled-components';
+import { WeatherData } from '../WeatherData/Index';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
 
 export function AdditionalWeatherData(props) {
-
   const { t } = useTranslation();
-  
+  const { currentForecast } = useSelector((state) => state.weather);
+  const { dew_point, humidity, visibility, uvi } = currentForecast;
+
+  const humidityFormatted = `${humidity}%`;
+  const dewPointFormatted = `${Math.round(dew_point)}°`;
+  const uvIndexFormatted = `${Math.round(uvi)}/10`;
+  const visibilityFormatted = `${(visibility / 1000).toFixed(1)}km`;
```

Comment:
Check out: https://www.i18next.com/translation-function/formatting & https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl
Most of the stuff you formatted here manually, you can do using the native `Intl` API and the `react-i18next` library (a must on big projects with multiple languages)

To be clear, this is ok for this project 😄